INIT.md

```diff
@@ -6,13 +6,13 @@
 A. **Local setup**
 
 Follow given steps to locally setup `webpack-cli init` by installing dependencies:
-1. Create `package.json` through npm
+1. Create `package.json` through npm.
```

Comment:
I believe these full-stops does not add any value. They are imperative statements just like Git Commit messages.